Description: ensure that the vaapi stubs are built against (a potentially custom) libva
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/media/gpu/vaapi/BUILD.gn
+++ b/media/gpu/vaapi/BUILD.gn
@@ -24,6 +24,7 @@ generate_stubs("libva_stubs") {
   }
   output_name = "va_stubs"
   deps = [ "//base" ]
+  configs = [ "//build/config/linux/libva" ]
 }
 
 source_set("vaapi") {
